packages/react-core/src/hooks/__tests__/useIsSignedIn.test.ts

```diff
@@ -0,0 +1,133 @@
+import useIsSignedIn from '../useIsSignedIn';
+import { act, renderHook } from '@testing-library/react-hooks';
+import { waitFor } from '@testing-library/react';
+import * as AuthModule from 'aws-amplify/auth';
+import { GetCurrentUserOutput } from 'aws-amplify/auth';
+import { Hub } from 'aws-amplify/utils';
+
+const getCurrentUserSpy = jest.spyOn(AuthModule, 'getCurrentUser');
+
+const mockError = new Error('Authorization error');
+const mockSuccess: GetCurrentUserOutput = {
```

Comment:
One thing @calebpollman had mentioned on my PRs is that these aren't really 'mocks', the mocks are the functions being mocked. I changed these names to `errorResult` and `successResult` or something like that.